docs/content/preview/integrations/atomicjar.md

```diff
@@ -0,0 +1,111 @@
+
+    <i class="fa-brands fa-centos"></i> [Docker for CentOS](https://store.docker.com/editions/community/docker-ce-server-centos)
+
+    <i class="fa-brands fa-ubuntu"></i> [Docker for Ubuntu](https://store.docker.com/editions/community/docker-ce-server-ubuntu)
+
+    <i class="icon-debian"></i> [Docker for Debian](https://store.docker.com/editions/community/docker-ce-server-debian)
+
+## Build the Testcontainer project
+
+Refer to the [Spring Boot source](https://github.com/yugabyte/yb-ms-data/tree/main/springboot) from GitHub for your local setup. The source consists of a basic JPA-based web application with CRUD functionality.
+
+You need to build the Testcontainer project using the following steps:
+
+1. Define the library dependency in the build file. Refer to the `build.grade` file for complete details.
+
+    ```conf
+
+    // version details for the BOM
+    extra["testcontainersVersion"] = "1.17.6"
+
+    // library
+    dependencies {
+        testImplementation("org.testcontainers:yugabytedb")
+        testImplementation("org.testcontainers:junit-jupiter")
+    }
+
+    dependencyManagement {
+    imports {
+        mavenBom("org.testcontainers:testcontainers-bom:${property("testcontainersVersion")}")
+    }
+    ```
+
+1. Initialize the YugabyteDB Testcontainer using the `@Container` annotation.
+
+    ```conf
+    @Container
+    YugabyteDBYSQLContainer container = new YugabyteDBYSQLContainer("yugabytedb/yugabyte:2.16.0.0-b90").withDatabaseName("yugabyte").withUsername("yugabyte").withPassword("yugabyte").withReuse(true);
+    ```
+
+   Refer to `AbstractTodoApplicationTest.java` class file for the complete definition.
+
+   The following snippet is required to populate the data source information:
+
+   ```conf
+   @DynamicPropertySource
+   static void datasourceProps(final DynamicPropertyRegistry registry) {
+     registry.add("spring.datasource.url", container::getJdbcUrl);
+     registry.add("spring.datasource.username", container::getUsername);
+     registry.add("spring.datasource.password", container::getPassword);
+     registry.add("spring.datasource.driver-class-name", () -> "com.yugabyte.Driver");
+   }
+   ```
+
+    An alternative to the preceding step is to define the `tc` JDBC URL in the `application-test.yaml` properties file.
+
+    ```conf
+    spring:
+      datasource:
+        url: jdbc:tc:yugabyte:2.14.4.0-b26:///yugabyte
+        username: yugabyte
+        password: yugabyte
+    ```
+
+## Write the integration test classes
+
+This example uses the standard way of writing integration test cases for the Spring Boot test framework with the following annotations:
+
+- `@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)` : This annotation loads the application context. The `webEnvironment` attribute is set to NONE because you do not want to start the embedded servlet container for your test. This property is mainly used when you want to test the service or repository layer.
```

Comment:
think i'd use dashes instead of colons
'this annotation' is redundant on all these bullets